Kinvey/Kinvey/ReadPolicy.swift

```diff
@@ -20,4 +20,7 @@ public enum ReadPolicy: UInt {
     /// Read first from the local cache and then try to get data from the network (backend).
     case both
     
+    /// Tries to get the most recent data from the backend first, if it fails it returns data from the local cache.
```

Comment:
Only should return data from the local cache if the server cannot be reached (i.e. does not return a result we can handle - 4xx/5xx).